magefiles/development.go

```diff
@@ -422,12 +422,17 @@ func patchDeployment(deployment *appsv1.Deployment, name string, container strin
 
 		if containerObj.Name == container {
 			containerObj.Image = image
-			// Set status reporting env variable to true.
+			// Set status reporting env variable to true. Also set Upgrade policy status reporting env variable to true.
+
 			containerObj.Env = []corev1.EnvVar{
 				{
 					Name:  "ENABLE_STATUS_REPORTING",
 					Value: "true",
 				},
+				{
+					Name:  "ENABLE_UPGRADEPOLICY_STATUS",
```

Comment:
We'll need to inject these env vars into the CI dynamically, we have a mage function which does the same, need to tweak it to include the above env var as well.

Ref: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/addon-operator/openshift-addon-operator-main.yaml#L126